themes/beaver/layouts/partials/icons/facebook.html

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg"
+aria-label="Facebook" role="img"
+viewBox="0 0 512 512"><path
+d="m0 0H512V512H0"
+fill="#fff"/><path d="m331 312h-45v134a192 192 0 10-60 0V312h-49v-56h49v-45q4-89 116-68v47h-26q-29 1-30 30v36h53.5" fill="#1877f2"/></svg>
```

Comment:
Could you add empty new line

Reply:
No need. Btw I converted to svg and use svg.html partial so you can preview svg in static/img/icons in your editor.